packages/backend/src/services/CsvService/CsvService.ts

```diff
@@ -781,6 +782,7 @@ export class CsvService extends BaseService {
                 exploreName: exploreId,
                 csvLimit,
                 context: QueryExecutionContext.CSV,
+                chartUuid: undefined,
```

Comment:
You can download CSV files from results, without saving a chart first. We generate the results from a metric query rather than a chart uuid. In particular, this method is triggered from the `{exploreId}/downloadCsv`  endpoint, so it might not have a chart uuid

I'm saying `might` because you can also trigger this endpoint from a saved chat uuid, that's why the chartName is there. We can update the query to also include the chartUuid on the `request`body as an optional parameter, for tracking purposes. 

Let me know if you think this is worth it.